macos/FluentUI/Button/Button.swift

```diff
@@ -228,16 +233,15 @@ open class Button: NSButton {
 		}
 	}
 
-	private var restBackgroundColor: NSColor {
-		return fillColor
-	}
-
+	@objc public var restBackgroundColor: NSColor? = nil
+	@objc public var cornerRadius: CGFloat = 3
```

Comment:
Because these are public, could we add some simple documentation around them? I realize the other variables do not have this.